### ISSUE TITLE
Two streams subscription in real time to different SubscribeToCharacteristic.

### ISSUE THREAD

**Issue by dawiddszewczyk** [PhilipsHue/flutter_reactive_ble#598]
**Describe the bug**
I would like to subscribe two services like "battery service " and "UART service" in realtime together to listen battery status and also get live data from device. But already I getting single live data.

**To Reproduce**
```  
Future<void> _onDeviceCallbackOne() async {
    _monitorLiveDataSubscription = RetryStream(() => flutterReactiveBle
            .subscribeToCharacteristic(QualifiedCharacteristic(characteristic))
            .doOnListen(() {
         print('Subscribed to LiveData');
        }).applyTransformer(transformer)).listen((event) {});

   _monitorLifeBatterySubscription = RetryStream(() => flutterReactiveBle
            .subscribeToCharacteristic(QualifiedCharacteristic(characteristicBattery))
            .doOnListen(() {
         print('Subscribed to Battery Life');
        }).applyTransformer(transformerBattery)).listen((event) {});
  }
```

**Smartphone **
 - Device: IPhone SE
 - OS: iOS 15
 - Package version: 5.0.2

**Comment by ElreboCM** [comment]
This seems to be a duplicate of Issue #526.

**Comment by ElreboCM** [comment]
@Taym95: Can you reason why you closed this issue? In my test, this issue is not solved yet (see also issue #526).

**Comment by Taym95** [comment]
duplicate issue
